test/unit/query/builder.js

```diff
@@ -2,21 +2,31 @@
 /*eslint no-var:0, indent:0, max-len:0 */
 'use strict';
 
-var MySQL_Client = require('../../../lib/dialects/mysql')
-var PG_Client = require('../../../lib/dialects/postgres')
-var Oracle_Client = require('../../../lib/dialects/oracle')
-var Oracledb_Client = require('../../../lib/dialects/oracledb')
-var SQLite3_Client = require('../../../lib/dialects/sqlite3')
-var MSSQL_Client = require('../../../lib/dialects/mssql')
+var _typeof2 = require('babel-runtime/helpers/typeof');
+
+var _typeof3 = _interopRequireDefault(_typeof2);
+
+var _keys = require('babel-runtime/core-js/object/keys');
```

Comment:
The answer is here: https://github.com/tgriesser/knex/pull/2063#issuecomment-301566939
Raw subqueries were always generated through Sqlite dialect, so changes in the Sqlite field wrapping actually broke other tests (like the one in the referenced comment)